### PR TITLE
Regenerated from OpenAPI spec

### PR DESCRIPTION
**Pull request by pietern**
## Changes

* Modification to template to regenerate the change made in #172 
* Use fully qualified type names instead of aliasing them on import (e.g. `compute.ComputeSpec`)
* Update integration tests to accommodate type name changes in the spec.

Spec: 8687781fe17d83694a0c67396e04c70830b0d398 (2023-06-23)

Type renames:
* `compute.BaseClusterInfo -> compute.ClusterSpec`
* `compute.ClusterInfo -> compute.ClusterDetails`
* `jobs.JobTaskSettings -> jobs.Task`
* `jobs.RunSubmitTaskSettings -> jobs.SubmitTask`
* `jobs.JobWebhookNotifications -> jobs.WebhookNotifications`
* `jobs.CreateJobFormat -> jobs.Format`
* `jobs.GitSourceGitProvider -> jobs.GitProvider`
* `jobs.FileArrivalTriggerSettings -> jobs.FileArrivalTriggerConfiguration`

## Tests

- [x] `make test` run locally
- [x] `make fmt` applied
- [x] relevant integration tests applied

### CODE REVIEW

File: .codegen/service.py.tmpl [hunk 1]
```diff
@@ -13,7 +13,7 @@ from ._internal import _enum, _from_dict, _repeated, Wait
 _LOG = logging.getLogger('databricks.sdk')
 
 {{range .ImportedEntities}}
-from .{{.Package.Name}} import {{.PascalName}}{{end}}
+from . import {{.Package.Name}}{{end}}
```
Comment (nfx) [inline .codegen/service.py.tmpl:16]: this will break `catalog`, `sharing`, and `jobs` with external imports, please revert

Comment (nfx) [reply .codegen/service.py.tmpl:16]: ah, i see - `'{{.Package.Name}}.{{.PascalName}}'`

can you change dot import to `from databricks.sdk.service import {{.Package.Name}}`

File: .codegen/service.py.tmpl [hunk 2]
```diff
@@ -44,8 +44,12 @@ class {{.PascalName}}{{if eq "List" .PascalName}}Request{{end}}:{{if .Descriptio
 {{end}}
 {{- define "from_dict_type" -}}
 	{{- if not .Entity }}None
-	{{- else if .Entity.ArrayValue }}{{if or .Entity.ArrayValue.IsObject .Entity.ArrayValue.IsExternal}}_repeated(d, '{{.Name}}', {{.Entity.ArrayValue.PascalName}}){{else}}d.get('{{.Name}}', None){{end}}
-	{{- else if or .Entity.IsObject .Entity.IsExternal }}_from_dict(d, '{{.Name}}', {{.Entity.PascalName}})
+	{{- else if .Entity.ArrayValue }}
```
Comment (nfx) [inline .codegen/service.py.tmpl:47]: can you do `{{template "type-nq" .Entity.ArrayValue}}`?

File: .codegen/service.py.tmpl [hunk 3]
```diff
@@ -275,7 +279,11 @@ self._api.do('{{.Verb}}', {{if .PathParts -}}
   {{- else if .Pagination -}}
     Iterator[{{.Pagination.Entity.PascalName}}]
   {{- else -}}
-    {{.Response.PascalName}}
+    {{- if .Response.IsExternal -}}
```
Comment (nfx) [inline .codegen/service.py.tmpl:282]: can you do `{{template "type-nq" .Response}}`?

**Review by nfx — CHANGES_REQUESTED** [review]
1. please regenerate the docs - `cd docs && make gen`
2. please add a list of type renames into PR - we'll consolidate in a release tag

File: .codegen/service.py.tmpl (hunk 1 above)
Comment (pietern) [reply .codegen/service.py.tmpl:16]: Done.

File: .codegen/service.py.tmpl (hunk 2 above)
Comment (pietern) [reply .codegen/service.py.tmpl:47]: Done, works for both cases below.

File: .codegen/service.py.tmpl (hunk 3 above)
Comment (pietern) [reply .codegen/service.py.tmpl:282]: If I do, then it yields the below diff. I believe we want to retain the current version where we use the listed type name.

```diff
--- a/databricks/sdk/service/settings.py
+++ b/databricks/sdk/service/settings.py
@@ -1164,7 +1164,7 @@ class WorkspaceConfAPI:
     def __init__(self, api_client):
         self._api = api_client

-    def get_status(self, keys: str, **kwargs) -> WorkspaceConf:
+    def get_status(self, keys: str, **kwargs) -> Dict[str, str]:
         """Check configuration status.

         Gets the configuration status for a workspace.
```

**Comment by pietern** [comment]
@nfx The docs update worked after all (had to reinstall the SDK package a second time for some reason...)

Included a list of type renames in the PR summary.

**Review by nfx — APPROVED** [review]

**Pull request merged**